lib/about_yml/schema.json

```diff
@@ -6,127 +6,127 @@
     "properties": {
         "name": {
             "type": "string",
-            "description": "Short name that acts as the project identifier"
+            "description": "This is a short name of your project that is used as a URL slug on the 18F dashboard."
```

Comment:
I think we should drop 18F-specific references throughout, or migrate them into the cheat sheet. Since we'd like to have other organizations consider adopting this standard, I'd like to not make it sound quite so 18F-centric.